src/components/container/RelativeActivity/RelativeActivity.tsx

```diff
@@ -39,7 +41,7 @@ export default function (props: RelativeActivityProps) {
             if (!results) return;
             let transformedResults: { [key: string]: RelativeActivityQueryResult } = {};
             dataTypes.forEach(dataType => {
-                if (results[dataType.dailyDataType]) {
+                if (results[dataType.dailyDataType]?.[getDayKey(date)]?.value) {
                     transformedResults[dataType.dailyDataType] = results[dataType.dailyDataType][getDayKey(date!)];
```

Comment:
Minor:  I believe you can remove the `!` from `date!` here and below as args to the `queryRelativeActivity(date!, date!, ...)` call since `date` will always populated.